src/tool_getparam.c

```diff
@@ -250,6 +250,9 @@ static const struct LongShort aliases[]= {
   {"Ek", "tlsuser",                  ARG_STRING},
   {"El", "tlspassword",              ARG_STRING},
   {"Em", "tlsauthtype",              ARG_STRING},
+#ifdef MPTCP
+  {"EM", "multipath",                ARG_BOOL},
+#endif  
```

Comment:
The cmdline options cannot be ifdef'ed like this, as the tool might not know what features the lib was built with.